yarn.lock

```diff
@@ -2353,7 +2353,7 @@
     ignore "^4.0.6"
     import-fresh "^3.2.1"
     js-yaml "^4.1.0"
-    minimatch "^3.0.5"
+    minimatch "^3.0.4"
```

Comment:
There are a lot of prescribed security patches in yarn.lock. You'll need to cherry pick only the changes relevant to your new dependency and revert the rest of the regressive updates.